extensions/control-plane/api/management-api/management-api-test-fixtures/src/testFixtures/java/org/eclipse/edc/connector/controlplane/test/system/utils/Participant.java

```diff
@@ -519,6 +521,11 @@ public void awaitTransferToBeInState(String transferProcessId, TransferProcessSt
         await().atMost(timeout).until(() -> getTransferProcessState(transferProcessId), it -> Objects.equals(it, state.name()));
     }
 
+    @Override
+    public String toString() {
+        return name;
+    }
+
```

Comment:
I generally discourage to override toString for "applicative" needs. `toString` method (according to docs):
> returns a string that "textually represents" this object

but the name does not represent it, let's use the `getName()` method directly